.github/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+#. Bug - Corrected EGAA inbounds to descend via BELZU - thanks to @ChrisXPP (Christoph Reule)
+
 # Changes from release 2023/09a to 2023/10
 1. Bug - Corrected Scottish TMA Mentor (STC_M_CTR) callsign prefix for alternative ownership functionality - thanks to @rishab-alt
 2. Enhancement - Added ground network data for Bristol (EGGD) - thanks to @MikePike-collab (Mike Pike) & @luke11brown (Luke Brown)
```

Comment:
```suggestion
# Changes from release 2023/09a to 2023/10
1. Bug - Corrected Scottish TMA Mentor (STC_M_CTR) callsign prefix for alternative ownership functionality - thanks to @rishab-alt
2. Enhancement - Added ground network data for Bristol (EGGD) - thanks to @MikePike-collab (Mike Pike) & @luke11brown (Luke Brown)
3. Bug - Corrected EGAA inbounds to descend via BELZU - thanks to @ChrisXPP (Christoph Reule)
```

If you do it like this (in the section for the next release with a number), it ensures it is either good to merge, or creates a conflict so we can fix it when merging. If you put the new line right at the top, it doesn't conflict I think. And the `#` is for section titles in the .md file, so just pop the number in is probably easiest. Ta :)